Frameworks/UIKit/UIView.mm

```diff
@@ -2252,12 +2251,16 @@ - (CGPoint)convertPoint:(CGPoint)pos fromView:(UIView*)fromView {
 */
 - (void)drawLayer:(CALayer*)layer inContext:(CGContextRef)context {
     UIGraphicsPushContext(context);
+    _CGContextPushBeginDraw(context);
```

Comment:
Er, that is, putting it into [CALayer display] broke things. I could go investigate if you'd like, but I'm not sure how fruitful that'd be - we'd probably have to keep this out of CALayer display anyway.

---
In reply to: [96095561](https://github.com/Microsoft/WinObjC/pull/1705#discussion_r96095561) [](ancestors = 96095561)